renovate.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": [
+    "config:js-app",
+    ":maintainLockFilesWeekly",
+    ":prNotPending",
+    ":unpublishSafe"
+  ],
+  "group": {
+    "commitMessage": "Update {{groupName}} packages",
+    "prTitle": "Update {{groupName}} packages"
+  },
+  "ignoreDeps": [
```

Comment:
These are all being ignored for now, since [bug 1364894](https://bugzilla.mozilla.org/show_bug.cgi?id=1364894) will be either upgrading them or removing them.